lib/pbench/test/unit/server/test_user_create.py

```diff
@@ -54,74 +61,125 @@ def test_args(server_config, pytestconfig):
-        runner = CliRunner(mix_stderr=False)
-        result = runner.invoke(
-            user_create.main,
-            args=[
-                TestCreateUser.USER_SWITCH,
-                TestCreateUser.USER_TEXT,
-                TestCreateUser.EMAIL_SWITCH,
-                TestCreateUser.EMAIL_TEXT,
-                TestCreateUser.FIRST_NAME_SWITCH,
-                TestCreateUser.FIRST_NAME_TEXT,
-                TestCreateUser.LAST_NAME_SWITCH,
-                TestCreateUser.LAST_NAME_TEXT,
-                TestCreateUser.ROLE_SWITCH,
-                "ADMIN",
-            ],
-            input=f"{TestCreateUser.PSWD_TEXT}\n",
-        )
-        assert result.exit_code == 0
-        assert (
-            result.stdout
-            == f"{TestCreateUser.PSWD_PROMPT}\n" + "Admin user test_user registered\n"
-        )
-        assert not result.stderr_bytes
+    def test_valid_user_delete(client, server_config, pytestconfig):
+        with client:
+            runner = CliRunner(mix_stderr=False)
+            TestUserManagement.test_args(server_config, pytestconfig)
+
+            result = runner.invoke(user_delete, args=[TestUserManagement.USER_TEXT,])
+            assert result.exit_code == 0
+            assert result.stdout == "User test_user deleted\n"
+            assert not result.stderr_bytes
 
     @staticmethod
     @responses.activate
-    def test_user_creation_with_invalid_role(server_config, pytestconfig):
-        runner = CliRunner(mix_stderr=False)
-        result = runner.invoke(
-            user_create.main,
-            args=[
-                TestCreateUser.USER_SWITCH,
-                TestCreateUser.USER_TEXT,
-                TestCreateUser.EMAIL_SWITCH,
-                TestCreateUser.EMAIL_TEXT,
-                TestCreateUser.FIRST_NAME_SWITCH,
-                TestCreateUser.FIRST_NAME_TEXT,
-                TestCreateUser.LAST_NAME_SWITCH,
-                TestCreateUser.LAST_NAME_TEXT,
-                TestCreateUser.ROLE_SWITCH,
-                "ADMN",
-            ],
-            input=f"{TestCreateUser.PSWD_TEXT}\n",
-        )
-        assert result.exit_code == 2
+    def test_valid_users_list(client, server_config, pytestconfig):
+        with client:
+            runner = CliRunner(mix_stderr=False)
+            TestUserManagement.test_args(server_config, pytestconfig)
+
+            result = runner.invoke(user_list,)
+            assert result.exit_code == 0
+            assert result.stdout == "{0:15}\t{1:15}\t{2:15}\t{3:15}\t{4:20}".format(
+                "Username", "First Name", "Last Name", "Registered On", "Email"
+            ) + "\n{0:15}\t{1:15}\t{2:15}\t{3:15}\t{4:20}\n".format(
+                "test_user", "First", "Last", "2021-04-16", "test@domain.com"
+            )
+            assert not result.stderr_bytes
+
+    @staticmethod
+    @responses.activate
+    def test_valid_users_update(client, server_config, pytestconfig):
```

Comment:
I think this test should now be named `test_invalid_user_update()` (singular "user").

Also, should there be a `test_invalid_user_update()`?